Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Views/LocalReceiptView.swift

```diff
@@ -84,6 +84,7 @@ private struct ReceiptDataView: View {
     init(receipt: AppleReceipt) {
         self.init(
             values: [
+                .init("Environment", (receipt.environment ?? .unknown).rawValue),
```

Comment:
Case in point :P we don't need optional + unknown

Reply:
Made it non-optional, with `.unknown` as default, and overridden by whatever comes in the receipt :D